### ISSUE TITLE
2.14.0 BugFixes or Bugs?

### ISSUE THREAD

**Issue by lilgandhi1199** [akiver/CSGO-Demos-Manager#563]
These are probably related to you fixing some analysis of certain demos and the next is probably another example of that related to round 16.
````markdown
"rounds_money_earned": {
"16": 4000,
changed to
"16": 3850,

"entry_kills": [
          {
            "round_number": 16,
            "killer_steamid": "76561198322402516",
            "killer_name": "Pogo $avag3 #ROAD TO SILVER 1",
            "killer_side": "T",
            "killed_steamid": "76561198068697969",
            "killed_name": "Lcuk",
            "killed_side": "CT",
            "weapon": {
              "element": 2,
              "type": 1,
              "weapon_name": "Glock-18"
            },
            "has_won": true,
            "has_won_round": false,
            "tick": 84566,
            "seconds": 1321.78809
          }

Changed to "has_won_round": true


Okay and then I can see that the team name's changed more accurately in round 16 probably
      "winner_side": "CT",
      "winner_name": "Team 2",
      "team_t_name": "Team 1",
      "team_ct_name": "Team 2",
````

But looking at
"most_headshot_player": {
    "$id": "2",
    "steamid": "76561198255320198",

  "most_headshot_player": {
    "$id": "2",
    "steamid": "76561198451236594",

In the new version it lists the same $id but a different steamID (in my case on a different team too) and player's JSON data.
I noticed you added most_damaging_weapon right above this. Did you just improve your HS % code?

**Comment by akiver** [comment]
Could you upload the affected demo?

**Comment by akiver** [comment]
About the HS% it's probably a bug and should be fixed with https://github.com/akiver/CSGO-Demos-Manager/pull/566

**Comment by akiver** [comment]
It should be fixed in 2.14.1, please open a new issue with affected demos if it's not the case.
